WcaOnRails/spec/requests/api_competitions_spec.rb

```diff
@@ -358,4 +390,4 @@
       end
     end
   end
-end
+end
```

Comment:
Could you configure your text editor to include newlines at the end of files? You'll see that Travis is complaining about this (and a number of other things) here: https://travis-ci.org/thewca/worldcubeassociation.org/builds/332609907#L866.

Reply:
I will look over this and the stuff Travis is complaining about.